### ISSUE TITLE
Cannon working on GALAH data with some attention needed to problems with scatter term. 

### ISSUE THREAD

**Issue by mkness** [mkness/TheCannon#61]
The Cannon nominally works on Galah data (using ccds 1-3 and not ccd 4)-  but there is a problem with the scatter term it is not optimising  -there is no minima in scatter at each pixel. Need to figure out why this is not working by plotting behaviour.. Also the errors in GALAH do not mask out skylines and other regions. The Galah results (attached below) will improve once the scatter problem is fixed I believe. 
![galah_spectra](https://cloud.githubusercontent.com/assets/937118/8546953/c64952ae-24bb-11e5-9724-084f223a82a4.png)
![4labels](https://cloud.githubusercontent.com/assets/937118/8546955/cd5882ea-24bb-11e5-8745-323e12c6ac3c.png)

**Comment by davidwhogg** [comment]
what do you mean "no minima"?  There MUST be a minimum...  Do you mean minimum is at zero?  If so, it *could* be that GALAH over-estimates its per-pixel noise variances.

**Comment by mkness** [comment]
Yes, as in the minima is 0 and yes this is entirely possible. I have just emailed to find out more details as to how the errors are determined.

**Comment by davidwhogg** [comment]
one sanity check:  Do your s/n estimates agree with GALAH's expectations?
